pkg/addon/parameters.go

```diff
@@ -43,26 +44,31 @@ func GetParameterByInstallation(ctx context.Context, client k8sclient.Client, in
 	)
 }
 
-// GetParameterByInstallType retrieves the value for an addon parameter by
-// selecting the addon name for installationType
+// GetParameterByInstallType retrieves the value for an addon parameter from
+// the addon parameters secret
 func GetParameterByInstallType(ctx context.Context, client k8sclient.Client, installationType integreatlyv1alpha1.InstallationType, namespace, parameter string) ([]byte, bool, error) {
-	addonName := GetName(installationType)
```

Comment:
I think this `GetName` function can now be removed from the code base. It does not provide a reliable way of getting the name of instance installed by the operator.